src/windows/find_tools.rs

```diff
@@ -199,6 +203,31 @@ mod impl_ {
         include: Vec<PathBuf>,
     }
 
+    fn is_amd64_emulation_supported() -> bool {
+        // In theory we should free the library handle, but since kernel32.dll is loaded in every
+        // process and we're a short-lived process, leaking the handles doesn't matter.
+        let kernel32 = unsafe { LoadLibraryA(b"kernel32.dll\0".as_ptr() as _) };
+        // GetMachineTypeAttributes is only available on Win11 22000+.
+        if let Some(get_machine_type_attributes) =
+            unsafe { GetProcAddress(kernel32, b"GetMachineTypeAttributes\0".as_ptr() as _) }
+        {
+            let mut attributes = Default::default();
+            let get_machine_type_attributes: unsafe extern "system" fn(
```

Comment:
Please split the type out as a type alias, it's a bit hard to read.

Reply:
Fair enough.